### PR TITLE
Update docs schema check

### PR DESCRIPTION
**Pull request by akito19**
This is document update to follow runners' updates, which are introducing schema checking.

### CODE REVIEW

File: tools/python/flake8.md [hunk 1]
```diff
@@ -38,11 +38,20 @@ linter:
       - flake8-mypy>=17.3.3
 ```
 
-### `version`
+### Options
+
+You can use several options to fine-tune stylelint to your project.
```
Comment (wata727) [inline tools/python/flake8.md]: Flake8 is not stylelint

Comment (akito19) [reply tools/python/flake8.md]: Oops, thank you for catching.

File: tools/swift/swiftlint.md [hunk 2]
```diff
@@ -18,20 +18,27 @@ Here are example settings for SwiftLint under `swiftlint`:
...
 
-### `ignore_warnings`
+### Options
 
-This option allows you to ignore issues that are `severity: "warnings"`. The default value is `false`.
+You can use several options to fine-tune SwiftLint to your project.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| [`ignore_warnigns`](#ignore_warnings) | `boolean` | If `true`, Sider will not report `severity: "warnings"` issues. |
```
Comment (wata727) [inline tools/swift/swiftlint.md]: `ignore_warnings`

File: tools/python/flake8.md [hunk 3]
```diff
@@ -38,11 +38,20 @@ linter:
...
 ```
 
-### `version`
+### Options
+
+You can use several options to fine-tune Flake8 to your project.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| [`version`](#version) | `integer` | Specify Python version when implementing `flake8`. |
```
Comment (wata727) [inline tools/python/flake8.md]: [IMO] I think it is better to describe "Python version" only.

Comment (wata727) [reply tools/python/flake8.md]: Probably, the description duplicates the below, so it should be concise.

**Pull request merged**